### DISCUSSION TITLE
Add option to exclude module from article pages

### DISCUSSION THREAD

**Discussion by QuotesUK** [joomla/joomla-cms#47060] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
When using Category Blog a module can be displayed on the blog layout and article pages for a selected category.

It isn’t possible to load a module on the blog layout only

Example scenario, using articles category module to display a list of articles with a specific tag/author etc to introduce the articles (and link to them) but not wanting that repeated on the article pages as well.

### Describe the solution you'd like
Display a module on the blog layout + don’t display on articles

Achieve this with the introduction of a global module option "show on article page" 

<img width="570" alt="articles-category" src="https://github.com/joomla/joomla-cms/assets/3391399/31c5a931-3aa0-451c-9268-9ee447e18a5a">

Alternatively there could be a new menu assignment option for "only on the parent pages selected"

<img width="703" alt="menu-assignment" src="https://github.com/joomla/joomla-cms/assets/3391399/6db05684-0b5c-4341-ab41-2cb7ae0de3a6">

### Additional context
Dynamic mode for articles category module has an option to exclude article pages. However this option isn’t available in normal mode or more generally on other module types.

If a module is loaded into the category description the menu assignment still controls whether it is displayed or not.

**Comment by DavidBoggitt** [comment]
Not sure if I'm allowed to post this here, but there are extensions that do this, for example:

https://regularlabs.com/advancedmodulemanager

**Comment by brianteeman** [comment]
This might help you https://brian.teeman.net/joomla/896-modules-nowhere

**Comment by rdeutz** [comment]
Would be something like:

1. On all pages
2. No pages
3. Only on the pages selected (with child pages)
4. Only on the pages selected (without childs pages)
5. On all pages except those selected (with child pages)
6. On all pages except those selected (without child pages)


1, 2, 3 & 5 would be like it is now
4 & 6 would be new options

So you can control when a module is displayed.

**Comment by carlitorweb** [comment]
I will make a draft and see from there
